controller/formation.go

```diff
@@ -182,8 +208,10 @@ func (r *FormationRepo) publish(appID, releaseID string) {
 	r.subMtx.RLock()
 	defer r.subMtx.RUnlock()
 
-	for ch := range r.subscriptions {
-		ch <- f
+	for sub := range r.subscriptions {
+		if !sub.notify(f) {
+			break
```

Comment:
We shouldn't break if one of the subscribers has gone away (the other subscribers may still be open)